changelogs/fragments/83803-collection-import-poll-interval.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+- >-
+  ``ansible-galaxy collection import`` - add a configuration option for the initial poll interval
```

Comment:
Sorry didn't pick this up, but I believe the command that calls this code is `publish`. The `import` one is for roles.